src/components/Profile.vue

```diff
@@ -1,10 +1,8 @@
 <template id="profile">
 
     <div class="profile-bar">
-        <div>Managing <router-link v-bind:to="'/user'"><strong>{{ selectedTrader.name }}</strong></router-link></div>
-        <transition name="fade" mode="out-in">
-            <div :key="voucherCount" v-if="vouchersAdded"><router-link v-bind:to="'/payment'"><strong>{{ voucherCount }}</strong> voucher<span v-if="voucherCount > 1">s</span> waiting</router-link></div>
-        </transition>
+        <div><router-link v-bind:to="'/account'"><strong>{{ selectedTrader.name }}</strong></router-link> <router-link v-bind:to="'/user'">(Change trader)</router-link></div>
+        <a v-on:click="onLogout">Log out</a>
     </div>
```

Comment:
There is definitely a lot of UI stuff in that corner - I think maybe it's something to look at in the next sprint. I'm happy with where everything is right now as that's how it is in the paper prototype.